src/cart/crt_context.c

```diff
@@ -1059,9 +1059,7 @@ crt_req_timeout_track(struct crt_rpc_priv *rpc_priv)
 	if (rc == 0) {
 		rpc_priv->crp_in_binheap = 1;
 	} else {
-		RPC_ERROR(rpc_priv,
-			  "d_binheap_insert failed, rc: %d\n",
-			  rc);
+		RPC_ERROR(rpc_priv, "d_binheap_insert failed, rc: %d\n", rc);
```

Comment:
May I suggest we start using DL_ERROR internally here (or add another version of RPC_ERROR that takes rc as a parameter).  It will print string version of error

Reply:
I dont think we ever hit this particular error; this would be something fatal.  I agree though we should use DL_ERROR in places where it makes sense